docs/user-guides/server.md

```diff
@@ -60,19 +60,19 @@ The procedure and UI of ONNX and TensorRT runtime would look the same as Pytorch
 
-Open AI has released 9 models so far. `ViT-B/32` is used as default model in all runtimes. Due to the limitation of some runtime, not every runtime supports all nine models. Please also note that different model give different size of output dimensions. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Here is a list of supported models of each runtime and its corresponding size:
+Open AI has released 9 models so far. `ViT-B/32` is used as default model in all runtimes. Due to the limitation of some runtime, not every runtime supports all nine models. Please also note that different model give different size of output dimensions. This will affect your downstream applications. For example, switching the model from one to another make your embedding incomparable, which breaks the downstream applications. Below is a list of supported models of each runtime and its corresponding size. We include the disk usage (in delta) and the peak RAM and VRAM usage (in delta) when running on a single Nvidia TITAN RTX GPU (24GB VRAM) when `batch_size=256`.
 
-| Model | PyTorch |  ONNX | TensorRT |  Output dimension | 
-| --- |---------| ---- | --- |--- |
-| RN50 | ✅ |✅ | ✅| 1024 | 
-| RN101 | ✅ |✅ | ✅| 512 | 
-| RN50x4 | ✅ |✅ | ✅| 640 |
-| RN50x16 | ✅ |✅ | ❌| 768 |
-| RN50x64 | ✅ |✅ | ❌| 1024 |
-| ViT-B/32  | ✅ |✅ | ✅| 512 |
-| ViT-B/16 | ✅ |✅ | ✅| 512 |
-| ViT-L/14 | ✅ |✅ | ✅| 768 |
-| ViT-L/14-336px | ✅ |✅ | ❌| 768 |
+| Model          | PyTorch | ONNX | TensorRT | Output Dimension | Disk Usage (MB) | Peak RAM Usage (GB) | Peak VRAM Usage (GB) |
+|----------------|---------|------|----------|------------------|-----------------|---------------------|----------------------|
+| RN50           | ✅       | ✅    | ✅        | 1024             | 256             | 3.44                | 4.25                 |
+| RN101          | ✅       | ✅    | ✅        | 512              | 292             | 3.51                | 4.28                 |
+| RN50x4         | ✅       | ✅    | ✅        | 640              | 422             | 4.14                | 6.92                 |
+| RN50x16        | ✅       | ✅    | ❌        | 768              | 661             | 5.07                | 13.01                |
+| RN50x64        | ✅       | ✅    | ❌        | 1024             | 1382            | 5.81                | 20.26                |
+| ViT-B/32       | ✅       | ✅    | ✅        | 512              | 351             | 3.66                | 2.19                 |
+| ViT-B/16       | ✅       | ✅    | ✅        | 512              | 354             | 3.58                | 3.90                 |
+| ViT-L/14       | ✅       | ✅    | ✅        | 768              | 933             | 4.11                | 5.38                 |
+| ViT-L/14-336px | ✅       | ✅    | ❌        | 768              | 934             | 5.07                | 11.36                |
```

Comment:
![image](https://user-images.githubusercontent.com/2041322/173507140-3f38d7f8-5733-4556-8be6-288dfc9a64af.png)

This number is impossible. In general, I believe the whole column is wrong.
- We run 2 `ViT-L/14-336px` on Berlin GPU for the last 2 months and together they take 5GB VRAM. 
- see also similar usage on Google Colab https://jina-ai.slack.com/archives/CTH1CMP8V/p1655126173106729
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/2041322/173507357-37f9b467-954a-475b-ae0a-23947fd76de4.png">

Reply:
In fact, the VRAM data is not wrong. I think the reason I got a large VRAM value is that I was using a mini-batch of size 256 in clip-server. If I stick to the default value, the VRAM indeed drops to ~4GB.

Reply:
I think I should clarify on that in the docs

Reply:
I see. Using a different batch size indeed introduces some confusion. Will update the table